hack/jenkins/upload-to-gcs.sh

```diff
@@ -115,11 +121,26 @@ function upload_artifacts_and_build_result() {
       echo "Uploading build log"
       gsutil -q cp -Z -a "${gcs_acl}" "${WORKSPACE}/build-log.txt" "${gcs_build_path}"
     fi
+
+    # For pull jobs, keep a canonical ordering for tools that want to examine
+    # the output.
+    echo "aoeu debug: ${gcs_indirect_path}"
```

Comment:
https://github.com/kubernetes/test-infra/pull/217

Once that goes in it should test your changes.